app/assets/stylesheets/nucore.scss

```diff
@@ -408,3 +438,191 @@ textarea.wide {
   padding: 0.4em;
   box-sizing: border-box;
 }
+
+/* Use/Manage CSS */
+body {
+  background: #ffffff; /* Old browsers */
+  background: -moz-linear-gradient(top, #ffffff 53%, #e4e4e4 100%); /* FF3.6-15 */
+  background: -webkit-linear-gradient(top, #ffffff 53%,#e4e4e4 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to bottom, #ffffff 53%,#e4e4e4 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#e4e4e4',GradientType=0 ); /* IE6-9 */
+}
+
+.manage-mode {
+  background: #ffffff; /* Old browsers */
+  background: -moz-linear-gradient(top, #ffffff 53%, $manageGradient 100%); /* FF3.6-15 */
+  background: -webkit-linear-gradient(top, #ffffff 53%,$manageGradient 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to bottom, #ffffff 53%, $manageGradient 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='$manageGradient',GradientType=0 ); /* IE6-9 */
+}
+
+html,body{
+  min-height: 100%;
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+body nav .navbar-inner .container .nav, .nav-tabs{
+  border-bottom: 1px solid $useColor;
+  min-height: 40px;
+}
+
+.manage-mode nav .navbar-inner .container .nav, 
+.manage-mode .nav-tabs {
+  border-color: $manageColorDark;
+}
+
+.alert-info {
+  background-color: transparent;
+  border-color: transparent;
+  color: $manageColorDark;
+  font-size: 24px;
+  line-height: 1em;
+}
+
+.navbar-inner {
+  background: none;
+  border: none;
+}
+
+.navbar-static-top .navbar-inner {
+  box-shadow: none;
+}
+
+nav ul.nav > li.active, nav  > 
+li.active, nav ul.nav > li:hover > 
+li {
+  border:none;
+  margin-top: 0;
+  margin-bottom: -1px;
+}
+
+nav ul.nav > li.active, nav  > 
+li.active, nav ul.nav > li:hover, nav > 
+li:hover {
+  border: none;
+  margin-top: 0;
+  background-color: white;
+}
+
+.container > .nav > li > a , .nav.nav-tabs > li > a{
+  padding: 11px 12px 11px 12px;
+  text-decoration: none;
+  color: $useColorDark;
+  transition: ease 0.3s all;
+}
+
+.manage-mode .container > .nav > li > a , .manage-mode .nav.nav-tabs > li > a{
+  color: $manageColorDark;
+}
+
+.container > .nav > li:hover > a {
+  text-shadow: 0 0 0.5px $useColorDark;
```

Comment:
Is it intentional to use the orange as the text-shadow color, even in manage mode?

<img width="201" alt="screen shot 2018-11-13 at 4 59 05 pm" src="https://user-images.githubusercontent.com/1099111/48448710-81c9cf80-e765-11e8-9261-885422419040.png">